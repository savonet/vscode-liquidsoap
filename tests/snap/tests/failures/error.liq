# Test uncaught errors.

def f() =
  error.raise("testing exceptions")
end

f()
