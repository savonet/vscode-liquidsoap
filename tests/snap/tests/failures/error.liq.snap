># Test uncaught errors.
#^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
>
>def f() =
#^^^ source.liquidsoap keyword.control.liquidsoap
#   ^^^^^^^ source.liquidsoap
>  error.raise("testing exceptions")
#^^^^^^^^^^^^^^ source.liquidsoap
#              ^ source.liquidsoap string.quoted.double.liquidsoap
#               ^^^^^^^^^^^^^^^^^^ source.liquidsoap string.quoted.double.liquidsoap
#                                 ^ source.liquidsoap string.quoted.double.liquidsoap
#                                  ^^ source.liquidsoap
>end
#^^^ source.liquidsoap keyword.control.liquidsoap
>
>f()
#^^^^ source.liquidsoap
>